conf/application.conf

```diff
@@ -85,9 +85,14 @@ controllers {
 }
 
 sendToActualRecipients = true
-additionalRecipients.email1 = "test@test"
-additionalRecipients.firstName1 = "ftest"
-additionalRecipients.lastName1 = "ltest"
+additionalRecipients = [
+  {
+    email: "test@test"
+    firstName: "ftest"
+    lastName: "ltest"
+    verified: false
```

Comment:
The previous config did not use `verified`. Can we default it to true without affecting functionality so that it doesn't have to be specified in `app-config-qa` etc?